core/src/test/java/org/kiwix/kiwixmobile/core/page/bookmark/viewmodel/effects/ShowDeleteBookmarksDialogTest.kt

```diff
@@ -47,8 +49,50 @@ internal class ShowDeleteBookmarksDialogTest {
     }
     val lambdaSlot = slot<() -> Unit>()
     showDeleteBookmarksDialog.invokeWith(activity)
-    verify { dialogShower.show(KiwixDialog.DeleteBookmarks, capture(lambdaSlot)) }
+    verify { dialogShower.show(any(), capture(lambdaSlot)) }
     lambdaSlot.captured.invoke()
     verify { effects.offer(DeleteBookmarkItems(effects, bookmarkState(), newBookmarksDao)) }
   }
+
+  @Test
+  fun `invoke with selected items shows dialog with DeleteSelectedBookmarks title`() {
+    val effects = mockk<PublishProcessor<SideEffect<*>>>(relaxed = true)
+    val newBookmarksDao = mockk<NewBookmarksDao>()
+    val activity = mockk<BookmarksActivity>()
+    val showDeleteBookmarksDialog =
+      ShowDeleteBookmarksDialog(
+        effects,
+        bookmarkState(listOf(bookmark(isSelected = true))),
+        newBookmarksDao
+      )
+    val dialogShower = mockk<DialogShower>()
+    every { activity.activityComponent.inject(showDeleteBookmarksDialog) } answers {
+      showDeleteBookmarksDialog.dialogShower = dialogShower
+      Unit
+    }
+    val lambdaSlot = slot<() -> Unit>()
+    showDeleteBookmarksDialog.invokeWith(activity)
+    verify { dialogShower.show(DeleteSelectedBookmarks, capture(lambdaSlot)) }
```

Comment:
we shouldn't use a capture slot if we aren't going to do anything with it, a shorthand for "idgaf about what this argument actually is" is to use the matcher `any()`